test/libyul/evmCodeTransform/eof_swap17.yul

```diff
@@ -0,0 +1,228 @@
+            sstore(17, v17)
+            sstore(16, v16)
+            sstore(15, v15)
+            sstore(14, v14)
+            sstore(13, v13)
+            sstore(12, v12)
+            sstore(11, v11)
+            sstore(10, v10)
+            sstore(9, v09)
+            sstore(8, v08)
+            sstore(7, v07)
+            sstore(6, v06)
+            sstore(5, v05)
+            sstore(4, v04)
+            sstore(3, v03)
+            sstore(2, v02)
+            sstore(1, v01)
+            sstore(0, v00)
+        }
+        f(
+            calldataload(0),
+            calldataload(1),
+            calldataload(2),
+            calldataload(3),
+            calldataload(4),
+            calldataload(5),
+            calldataload(6),
+            calldataload(7),
+            calldataload(8),
+            calldataload(9),
+            calldataload(10),
+            calldataload(11),
+            calldataload(12),
+            calldataload(13),
+            calldataload(14),
+            calldataload(15),
+            calldataload(16),
+            calldataload(17)
+        )
+	}
+}
+// ====
+// EVMVersion: >=osaka
+// bytecodeFormat: >=EOFv1
+// stackOptimization: true
+// ----
+//     /* "":1172:1174   */
+//   0x11
+//     /* "":1159:1175   */
+//   calldataload
+//     /* "":1142:1144   */
+//   0x10
+//     /* "":1129:1145   */
+//   calldataload
+//     /* "":1112:1114   */
+//   0x0f
+//     /* "":1099:1115   */
+//   calldataload
+//     /* "":1082:1084   */
+//   0x0e
+//     /* "":1069:1085   */
+//   calldataload
+//     /* "":1052:1054   */
+//   0x0d
+//     /* "":1039:1055   */
+//   calldataload
+//     /* "":1022:1024   */
+//   0x0c
+//     /* "":1009:1025   */
+//   calldataload
+//     /* "":992:994   */
+//   0x0b
+//     /* "":979:995   */
+//   calldataload
+//     /* "":962:964   */
+//   0x0a
+//     /* "":949:965   */
+//   calldataload
+//     /* "":933:934   */
+//   0x09
+//     /* "":920:935   */
+//   calldataload
+//     /* "":904:905   */
+//   0x08
+//     /* "":891:906   */
+//   calldataload
+//     /* "":875:876   */
+//   0x07
+//     /* "":862:877   */
+//   calldataload
+//     /* "":846:847   */
+//   0x06
+//     /* "":833:848   */
+//   calldataload
+//     /* "":817:818   */
+//   0x05
+//     /* "":804:819   */
+//   calldataload
+//     /* "":788:789   */
+//   0x04
+//     /* "":775:790   */
+//   calldataload
+//     /* "":759:760   */
+//   0x03
+//     /* "":746:761   */
+//   calldataload
+//     /* "":730:731   */
+//   0x02
+//     /* "":717:732   */
+//   calldataload
+//     /* "":701:702   */
+//   0x01
+//     /* "":688:703   */
+//   calldataload
+//     /* "":672:673   */
+//   0x00
+//     /* "":659:674   */
+//   calldataload
+//     /* "":644:1185   */
+//   callf{code_section_1}
+//     /* "":22:1188   */
+//   stop
+//
+// code_section_1: assembly {
+//         /* "":29:635   */
+//       swapn{17}
```

Comment:
This is actually stupid code to generate (instead of first shuffling the entire stack, it should start consuming with ``sstore``s immediately) - and we'll get better code here with the SSA-based code transforms I hope - but in any case, it's impossible to generate code without a ``swapn{17}`` here.